### PR TITLE
Add layer odbc

### PR DESCRIPTION
**Pull request by ondrajodas**

### CODE REVIEW

**Comment by Nyholm** [comment]
Thank you. 

Im not familiar with odbc. Could you tell me what it is? 
Also, it does not look like a PHP extension. How do I use this?

**Comment by ondrajodas** [comment]
@Nyholm ODBC is php extension for connect to database eg. `Snowflake database`
this function is described here:
https://www.php.net/manual/en/function.odbc-connect.php

File: layers/odbc/Dockerfile [hunk 1]
```diff
@@ -0,0 +1,16 @@
+ARG PHP_VERSION
+FROM bref/build-php-$PHP_VERSION AS ext
+
+ENV LD_LIBRARY_PATH=/usr/lib:/usr/lib64:$LD_LIBRARY_PATH
+
+# Install packages
+RUN yum install -y php-odbc
```
Comment (Nyholm) [inline layers/odbc/Dockerfile]: Aren’t you missing a pecl install?

Comment (ondrajodas) [reply layers/odbc/Dockerfile]: i can't install odbc via pecl, because this ended with error

Comment (ondrajodas) [reply layers/odbc/Dockerfile]: image

**Comment by Nyholm** [comment]
With my little understanding, it does not seam that till will allow you to use odbc functions with php with this PR. You only included the library and no way for php to use that library.

**Comment by ondrajodas** [comment]
@Nyholm ok i looking on it

**Comment by ondrajodas** [comment]
I'm no longer need, so i close this PR. thanks for your time

**Pull request closed without merging**